package.json

```diff
@@ -12,10 +12,11 @@
     "@connectrpc/connect": "^1.4.0",
     "@penumbra-zone/bech32m": "^7.0.0",
     "@penumbra-zone/client": "^18.0.0",
-    "@penumbra-zone/getters": "^15.0.0",
+    "@penumbra-zone/getters": "^16.0.0",
     "@penumbra-zone/protobuf": "^6.0.0",
     "@penumbra-zone/transport-dom": "^7.5.0",
-    "@penumbra-zone/types": "^20.0.0",
+    "@penumbra-zone/types": "^21.0.0",
+    "@penumbra-zone/ui": "file:../web/packages/ui/penumbra-zone-ui-8.1.0.tgz",
```

Comment:
issue(blocking): waiting till publish?

Reply:
yep, waiting for https://github.com/penumbra-zone/web/pull/1733

Reply:
updated the deps